<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>740</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>740</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>740</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1200</width>
    <height>740</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">/*background-color: rgba(54, 30, 61, 1);
background-image:url(:images/theme/bg1/jpg);*/
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="desk" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>900</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0.5)</string>
    </property>
   </widget>
   <widget class="QWidget" name="sidebar" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>300</width>
      <height>740</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="logo">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>10</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../images.qrc">:/new/theme/images/theme/pharos_white.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="toolbar" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>300</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgba(0,0,0,0.5);</string>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="friendButton">
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../images.qrc">
           <normaloff>:/new/buttons/images/buttons/friend.png</normaloff>:/new/buttons/images/buttons/friend.png</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="roomButton">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../images.qrc">
           <normaloff>:/new/buttons/images/buttons/chat.png</normaloff>:/new/buttons/images/buttons/chat.png</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addFriendButton">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../images.qrc">
           <normaloff>:/new/buttons/images/buttons/addFriend.png</normaloff>:/new/buttons/images/buttons/addFriend.png</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addRoomButton">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../images.qrc">
           <normaloff>:/new/buttons/images/buttons/addChat.png</normaloff>:/new/buttons/images/buttons/addChat.png</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="addressList" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>300</width>
       <height>580</height>
      </rect>
     </property>
     <widget class="QListWidget" name="roomAddressList">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>2</y>
        <width>300</width>
        <height>550</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>方正喵呜体</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
     </widget>
     <widget class="QListWidget" name="friendAddressList">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>2</y>
        <width>300</width>
        <height>580</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>方正喵呜体</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget{
background-color:transparent;
};

QListWidget::item:hover{
background-color: rgba(0,0,0,0.5);
};
 
QListWidget::item:selected{
background-color: rgba(0,0,0,0.8);
};</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="searchInput">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>700</y>
       <width>210</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(256, 256, 256, 0.9); 
border-radius: 5px; 
border: 1px solid rgba(0, 0, 0, 0.5);
color: white;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>695</y>
       <width>40</width>
       <height>35</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/new/buttons/images/buttons/search.png</normaloff>:/new/buttons/images/buttons/search.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="side_clientName">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线 Light</family>
       <pointsize>19</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white; background-color: rgba(0,0,0,0); border-left: 1px solid white; padding-left: 2px;</string>
     </property>
     <property name="text">
      <string>Grayson</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
    <widget class="QLabel" name="logo_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgba(0,0,0,0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../images.qrc">:/new/theme/images/theme/name.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>740</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images.qrc">:/new/theme/images/theme/bg1.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>0</y>
      <width>155</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="titlebar">
     <item>
      <widget class="QPushButton" name="title_settingButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/new/buttons/images/buttons/setting.png</normaloff>:/new/buttons/images/buttons/setting.png</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="title_backwardButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/new/buttons/images/buttons/backward.png</normaloff>:/new/buttons/images/buttons/backward.png</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="title_forwardButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/new/buttons/images/buttons/forward.png</normaloff>:/new/buttons/images/buttons/forward.png</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="title_exitButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/new/buttons/images/buttons/exit.png</normaloff>:/new/buttons/images/buttons/exit.png</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>title_backwardButton</zorder>
    <zorder>title_settingButton</zorder>
    <zorder>title_exitButton</zorder>
    <zorder>title_forwardButton</zorder>
   </widget>
   <widget class="QPushButton" name="title_informButton">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>6</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/new/buttons/images/buttons/newInform.png</normaloff>:/new/buttons/images/buttons/newInform.png</iconset>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>sidebar</zorder>
   <zorder>desk</zorder>
   <zorder>horizontalLayoutWidget_2</zorder>
   <zorder>title_informButton</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
